docs/software/switch-config.md

```diff
@@ -8,6 +8,46 @@ description: To optimize the performance of ExpressLRS, it has different Switch
 !!! warning "WARNING" 
     **Put your arm switch on AUX1**, and set it as **~1000 is disarmed, ~2000 is armed**.
 
+# Summary of Switch Configs
+
+This table summarizes the switch configuration modes and the available switch positions / resolution and update frequency on each channel or flight controller auxiliary channel (Aux X).
+
+| Channel | Flight  <br>Controller | Hybrid | Wide  <br>Hybrid | Full Res  <br>8 | Full Res  <br>16 Half Rate | Full Res  <br>12 Mixed |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| 1   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 2   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 3   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 4   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| **5<sup>+</sup>**   | Aux 1 | **2-pos** | **2-pos** | **2-pos** | **2-pos** | **2-pos** |
+| 6   | Aux 2 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 7   | Aux 3 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 8   | Aux 4 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 9   | Aux 5 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 10  | Aux 6 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 11  | Aux 7 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 12  | Aux 8 | *16-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 13  | Aux 9 | - | - | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 14  | Aux 10 | - | - | - | Full Res<sup>H</sup> | - |
+| 15  | Aux 11 | - | - | - | Full Res<sup>H</sup> | - |
+| 16  | Aux 12 | - | - | - | Full Res<sup>H</sup> | - |
+ 
+**Switch Position / Channel Resolution**<br>
+**Full Res** = 10-bit or 1024 positions for full resolution for a PWM between 988 to 2012<br>

```

Comment:
This is incorrect about `Full Res`. There's two options here but I'm not sure of the best way to shrink it down
Normal Modes 10 bit (everything except FullRes modes): 10 bit, 988-2012us. 1 bit = 1us
![range-normal](https://user-images.githubusercontent.com/677183/184015334-1a436e22-d660-491a-adcf-d7e721fed31f.png)

Full Res 10 bit (CRSF Extended Limits): 10 bit, 885-2115us, 1 bit = 1.23046875us
![range-crsf](https://user-images.githubusercontent.com/677183/184015610-1a04ce51-6640-4742-a41d-8cd4a52b6056.png)

Reply:
Ok I think I got it figured out. See the updated table. I decided not to worry about the graphical images of the ranges yet. I'll do that at a later point when I get more experience in the elrs project.